### DISCUSSION TITLE
axle for full power shift transmission

### DISCUSSION THREAD

**Discussion by astro80** [Mogli12/KeyboardSteer#739] [Previous version]
hello it`s possible to have an axis for gear shifting  to simulate FPS Transmissions?
thanks for your wonderful work!

**Comment by Mogli12** [comment]
I think you can always assign a half axis to normal key binding.

**Comment by astro80** [comment]
I try this with an axis of my hotas, but i`m able to shift only one gear up or down

**Comment by Mogli12** [comment]
Yes, what do you expect?
